packages/cactus-cmd-api-server/src/main/json/openapi.json

```diff
@@ -128,6 +132,31 @@
                     }
                 }
             }
+          },
+          "/api/v1/api-server/get-open-api-spec-v1-endpoint": {
+            "post": {
+                 "description": "Returns the openapi.json document of specific plugin.",
+                 "x-hyperledger-cactus": {
+                     "http": {
+                         "verbLowerCase": "get",
+                         "path": "/api/v1/api-server/get-open-api-spec-v1-endpoint"
+                     }
+                 },
+                  "operationId": "getOpenApiSpec",
```

Comment:
@charellesandig Please include the major  version number in the name so that once we do more major releases we can more easily maintain the different versions in parallel. Please see the other names for the existing naming convention to mimic. (the ones where the convention does exist - unfortunately not all names)